pkg/cli/environment.go

```diff
@@ -91,5 +91,8 @@ func (s *EnvSettings) EnvVars() map[string]string {
 		"HELM_REGISTRY_CONFIG":   s.RegistryConfig,
 		"HELM_REPOSITORY_CACHE":  s.RepositoryCache,
 		"HELM_REPOSITORY_CONFIG": s.RepositoryConfig,
+		"HELM_NAMESPACE":         s.Namespace,
```

Comment:
The name `HELM_NAMESPACE` is borrowed form an another PR from @adamreese https://github.com/helm/helm/pull/6508/files#r330893957

Other two are named such so that they become consistent with all the other envvars.